### PR TITLE
Random Values 0.1.0

### PR DESCRIPTION
**Pull request by sidewaysthought**
I added a plugin that provides the following utilities

- make_uuids generates 1 or more UUIDs (128-bit label)
- generate_string generates 1 or more alphanumeric strings of at least 2 characters in length
- generate_password generates 1 or more passwords of 6 or more characters using letters, numbers and punctuation
- generate_placeholder_text generates 1 or more sentences of lorem ipsum text
- random_number draws 1 or more random numbers between min and max

### CODE REVIEW

File: src/autogpt_plugins/random_values/__init__.py [hunk 1]
```diff
@@ -0,0 +1,240 @@
+"""Wikipedia search integrations."""
```
Comment (ntindle) [inline src/autogpt_plugins/random_values/__init__.py:1]: Fix

File: src/autogpt_plugins/random_values/__init__.py [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+
+class Message(TypedDict):
+    role: str
+    content: str
+
+
+class AutoGPTRandomValues(AutoGPTPluginTemplate):
+    """
+    Wikipedia search integrations
```
Comment (ntindle) [inline src/autogpt_plugins/random_values/__init__.py:21]: Fix

**Comment by desojo** [comment]
It looks like there are some typing errors when AutoGPT doesn't pass the right datatype. Since python can handle type conversion itself, make the args type agnostic

```
Input:generate a random number
SYSTEM:  Human feedback: generate a random number
 THOUGHTS:  I will generate a random number for the user.
REASONING:  As an AI, I can perform simple tasks like generating random numbers for the user. This will help me to be useful to the user and provide value.
PLAN: 
-  Generate a random number for the user
CRITICISM:  I need to ensure that the random number I generate is truly random and that I am not generating the same number multiple times.
NEXT ACTION:  COMMAND = random_number ARGUMENTS = {'min': '1', 'max': '100', 'count': '1'}
  Enter 'y' to authorise command, 'y -N' to run N continuous commands, 's' to run self-feedback commands'n' to exit program, or enter feedback for ...
  Asking user via keyboard...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-= 
SYSTEM:  Command random_number returned: Error: '<=' not supported between instances of 'int' and 'str'
 THOUGHTS:  I will generate a random number for the user.
REASONING:  As an AI, I can perform simple tasks like generating random numbers for the user. This will help me to be useful to the user and provide value.
PLAN: 
-  Generate a random number for the user
CRITICISM:  I need to ensure that the random number I generate is truly random and that I am not generating the same number multiple times.
NEXT ACTION:  COMMAND = random_number ARGUMENTS = {'min': 1, 'max': 100, 'count': 1}
  Enter 'y' to authorise command, 'y -N' to run N continuous commands, 's' to run self-feedback commands'n' to exit program, or enter feedback for ...
  Asking user via keyboard...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-= 
SYSTEM:  Command random_number returned: [11]
```

**Review by desojo — CHANGES_REQUESTED** [review]
See comments

**Review by desojo — APPROVED** [review]
Test output for:

Random Number:
```
NEXT ACTION:  COMMAND = Random Number ARGUMENTS = {'min': '0', 'max': '100', 'count': '10'}
  Enter 'y' to authorise command, 'y -N' to run N continuous commands, 's' to run self-feedback commands'n' to exit program, or enter feedback for ...
  Asking user via keyboard...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-= 
SYSTEM:  Command Random Number returned: [4, 77, 33, 84, 20, 38, 24, 61, 12, 14]
```

UUID
```
NEXT ACTION:  COMMAND = make_uuids ARGUMENTS = {'count': '5'}
  Enter 'y' to authorise command, 'y -N' to run N continuous commands, 's' to run self-feedback commands'n' to exit program, or enter feedback for ...
  Asking user via keyboard...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-= 
SYSTEM:  Command make_uuids returned: ["4116864c-7fe6-4d9d-82df-14563c047994", "7e989c21-baf0-43b1-962f-cabc5fb9ab3d", "21dbef81-d657-4546-8d89-11ed6272902d", "babe2044-0285-4769-8628-e662dab4f98b", "54dac78b-f1f4-44b5-ba4e-6588529fd200"]
```

Generate String
```
NEXT ACTION:  COMMAND = generate_string ARGUMENTS = {'length': '10', 'count': '5'}
  Enter 'y' to authorise command, 'y -N' to run N continuous commands, 's' to run self-feedback commands'n' to exit program, or enter feedback for ...
  Asking user via keyboard...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-= 
SYSTEM:  Command generate_string returned: ["pxUxqyGJOJ", "ciUQskmOqU", "xBJKlTPmXo", "KYqUURUHFZ", "XUYvcvBdhu"]
```

Generate Password:
```
NEXT ACTION:  COMMAND = generate_password ARGUMENTS = {'length': '12', 'count': '1'}
  Enter 'y' to authorise command, 'y -N' to run N continuous commands, 's' to run self-feedback commands'n' to exit program, or enter feedback for ...
  Asking user via keyboard...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-= 
SYSTEM:  Command generate_password returned: ["So7hmit_@lpl"]
```

Generate Placeholder
```
NEXT ACTION:  COMMAND = generate_placeholder_text ARGUMENTS = {'sentences': '5'}
  Enter 'y' to authorise command, 'y -N' to run N continuous commands, 's' to run self-feedback commands'n' to exit program, or enter feedback for ...
  Asking user via keyboard...
Input:y
-=-=-=-=-=-=-= COMMAND AUTHORISED BY USER -=-=-=-=-=-=-= 
SYSTEM:  Command generate_placeholder_text returned: ["Mollit aliquip consequat officia.", "Nostrud exercitation anim sint dolor, reprehenderit enim cillum sed proident ut velit.", "Laborum fugiat veniam est quis.", "Aute ullamco reprehenderit sit mollit amet.", "Ea non officia ut aliqua aliquip exercitation nisi, lorem incididunt ex ipsum."]
```

Success!

**Pull request merged**